problems/removeNumbersAtOddIndices.js

```diff
@@ -2,8 +2,7 @@
 * @param {number[]} arr - The input array
 * @returns {number[]} - An array removing all elements initially appearing at an odd index
 */
-function removeNumbersAtOddIndices() {
-
-}
+function removeNumbersAtOddIndices(arr) {
```

Comment:
Try using filter for this one as well. Remember the callback on array methods can also take the index as a second argument.